exchanges/kraken/kraken_wrapper.go

```diff
@@ -506,7 +504,7 @@ func (k *Kraken) UpdateOrderbook(ctx context.Context, p currency.Pair, assetType
 				Amount: futuresOB.Orderbook.Bids[y][1],
 			}
 		}
-		book.Bids.Reverse()
+		book.Bids.SortBids() // Kraken returns bids in descending or ascending order
```

Comment:
Somewhat OOS comment (doesn't make sense with our current priority of short linecount > speed for REST), but if those are the only two possibilities we could do something like, if len > 2 && elementOnePrice < elementTwoPrice, reverse slice. Otherwise, just keep it.

Something to keep in mind for when we move into the optimisation phase.

Reply:
Yeah. I think this is one of those comments that belongs in a commit message, but not in the code.
i.e. Once it's there, there will be no need to explain it. Future readers will just be like "erm. Okay."
```suggestion
		book.Bids.SortBids()
```